primitive/compute/description/fully_specified.go

```diff
@@ -42,6 +42,61 @@ func MarshalSteps(step *pipeline.PipelineDescription) (string, error) {
 	return string(stepJSON), nil
 }
 
+// CreateImageQueryPipeline creates a pipeline that will perform image retrieval.
+func CreateImageQueryPipeline(name string, description string) (*FullySpecifiedPipeline, error) {
+	steps := []Step{
+		NewDatasetToDataframeStep(map[string]DataRef{"inputs": &PipelineDataRef{0}}, []string{"produce"}),
+		NewRemoveColumnsStep(
+			map[string]DataRef{"inputs": &StepDataRef{0, "produce"}},
+			[]string{"produce"},
+			[]int{1, 2, 3, 4},
```

Comment:
Can we do this without hardcoding?  I think elsewhere we ran an `ExtractByStructuralType` after column parsing, pulling out only those features that were numerical values.  Its a slightly more flexible subsitute for the proper solution, which is having the pre-featurization step assign a semantic type of "compute_feature" (or similar) when it creates the image vector, and then keying off of that.